addons/website/static/tests/tours/edit_menu.js

```diff
@@ -0,0 +1,114 @@
+/** @odoo-module */
+
+import wTourUtils from 'website.tour_utils';
+import tour from 'web_tour.tour';
+
+const clickOnExtraMenuItem = {
+    content: "Click on the extra menu modal if it is there",
+    trigger: '#top_menu',
+    run: function () {
+        const extraMenuButton = this.$anchor[0].querySelector('.o_extra_menu_items a.nav-link');
+        if (extraMenuButton) {
+            extraMenuButton.click();
+        }
+    }
+};
+
+const clickOnSave = {
+   content: "Clicks on the menu edition dialog save button",
+   trigger: '.modal-dialog .btn-primary span:contains("Save")',
+};
+
+tour.register('edit_menu', {
+    test: true,
+    url: '/',
+}, [
+    // Add a megamenu item from the menu.
+    {
+        content: "Open Pages menu",
+        trigger: '.o_menu_sections a:contains("Pages")',
+    },
+    {
+        content: "Click on Edit Menu",
+        trigger: 'a.dropdown-item:contains("Edit Menu")',
+    },
+    {
+        content: "Trigger the link dialog (click 'Add Mega Menu Item')",
+        extra_trigger: '.o_web_editor_dialog:visible',
+        trigger: '.modal-body a.js_add_menu[data-type="mega"]',
+    },
+    {
+        content: "Write a label for the new menu item",
+        extra_trigger: '.o_link_dialog',
+        trigger: '.o_link_dialog #o_link_dialog_label_input',
+        run: 'text Megaaaaa!'
+    },
+    clickOnSave,
+    clickOnSave,
+    {
+        ...clickOnExtraMenuItem,
```

Comment:
`...` inside an object is not something with support, change it to `Object.assign`